modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-openstack-ts-az-config_{context}"]
+= Configuring {rh-openstack} Availability Zones type
+
+You have installed a cluster on {rh-openstack-first} or on existing {product-title} cluster version 4.13 or lower, and the root volumes are deployed without updating Availability Zones (AZ) in the `install-config.yaml` file. 
+
+This makes the root volume availability zone to be taken from the compute availability zone, which is not available in cinder. The control plane machine cannot be created. 
+
+[NOTE]
+====
+For the {product-title} clusters version 4.14 and higher, the installers validate the root AZ in `install-congif.yaml` file and ensures the deployment to be continued.
+====
+
+For the clusters deployed before the {product-title} version 4.14, you need to manually update the control plane machine resources, so that they are updated on the instances.
+
+[NOTE]
+====
+Manually editing control plane machine resources does not update the control plane instances. However, it is necessary in order for the `cluster-control-plane-machine-set-operator` to correctly generate a `ControlPlaneMachineSet` for your cluster.
```

Comment:
If something is necessary, it should be part of the main text or else boxed off as `IMPORTANT`.

Reply:
It was actually an additional info. Let me add this as an [IMPORTANT] text.